apps/full-site-editing/full-site-editing-plugin/dotcom-fse/blocks/navigation-menu/edit.js

```diff
@@ -50,22 +50,25 @@ const NavigationMenuEdit = ( {
 				/>
 			</BlockControls>
 			<InspectorControls>
-				<PanelBody className="blocks-font-size" title={ __( 'Text Settings' ) }>
+				<PanelBody
+					className="blocks-font-size"
+					title={ __( 'Text Settings', 'full-site-editing' ) }
+				>
 					<FontSizePicker onChange={ setFontSize } value={ actualFontSize } />
 				</PanelBody>
 				<PanelColorSettings
-					title={ __( 'Color Settings' ) }
+					title={ __( 'Color Settings', 'full-site-editing' ) }
```

Comment:
Is there a way to shim `__` so that it always has the correct text domain in the FSE plugin? The main thing I can think of would be pretty hacky or maybe too much work/refactor. What do folks normally do in these situations?

Reply:
We're actually rather considering to introduce an eslint rule to enforce it. What do you think?

Reply:
That sounds good to me. 👍